Source/WebKit/UIProcess/Cocoa/WebPageProxyCocoa.mm

```diff
@@ -1085,7 +1085,7 @@ static bool exceedsRenderTreeSizeSizeThreshold(uint64_t thresholdSize, uint64_t
 
 bool WebPageProxy::useGPUProcessForDOMRenderingEnabled() const
 {
-    if (id useGPUProcessForDOMRendering = [[NSUserDefaults standardUserDefaults] objectForKey:@"WebKit2GPUProcessForDOMRendering"])
+    if (RetainPtr<id> useGPUProcessForDOMRendering = [[NSUserDefaults standardUserDefaults] objectForKey:@"WebKit2GPUProcessForDOMRendering"])
```

Comment:
It’s possible that just `RetainPtr` will work here and in other places like it after @ddkilzer’s recent `adoptNS<id>` fix.